Dockerfile-swagger-codegen-python

```diff
@@ -0,0 +1,11 @@
+FROM openjdk:8-jre-alpine
```

Comment:
Instead of having multipe Docker-swagger-codegen-* files, it would be better to have a generic one that accept `LANGUAGE` as a build argument (https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg).

Reply:
The problem here is that we have different commands for each of the docker files. For python docker file we need to have "python-config.json" and we need to use it with option '-c'.

As far as I know, making this with build arguments is not possible. Instead I made changes "build-swagger-codegen.sh" that makes it accept two arguments "v for version and l for language".

Reply:
Ok, let's keep it like that.